saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -489,6 +494,22 @@ internal void SetRadioChannel(string radioChannelName, bool primary)
 
             BaseScript.TriggerServerEvent(Event.SaltyChat_SetRadioChannel, radioChannelName, primary);
         }
+
+        internal void SetRadioVolume(float radioVolumeNewValue)
+        {
+            if (radioVolumeNewValue < 0.10)
+            {
+                this.RadioVolume = 0.1f;
+            }
+            else if (radioVolumeNewValue > 1.60)
+            {
+                this.RadioVolume = 1.6f;
+            }
+            else
+            {
+                this.RadioVolume = radioVolumeNewValue;
+            }
```

Comment:
Since there is only one line in each if/else:
```csharp
if (volumeLevel < 0f)
    this.RadioVolume = 0f;
else if (volumeLevel > 1.6f)
    this.RadioVolume = 1.6f;
else
    this.RadioVolume = volumeLevel;
```